### DISCUSSION TITLE
Track "all contributors"?

### DISCUSSION THREAD

**Discussion by wjbenfold** [SciTools/iris#4680] [Ideas]
@bjlittle recently introduced me to [all-contributors](https://allcontributors.org/docs/en/overview) as a way of recognising contributions to a repository that aren't just authoring commits.

We could set it up so that their bot is watching GitHub and then we'd be able to recognise people using something like

> @all-contributors please add @wjbenfold for code

You can get the bot to put up a pull request so that the person has an opportunity to ask not to be added. We could then put the table of the information in the repo's README (which is currently pretty short), or keep the README clean and tidy by adding a page to the docs (though that might get fewer views)?

I'd suggest that the following categories would probably be relevant to us (though I think you just get them all available, including those that aren't below, as options):

(🐛 bug) Bug reports
(💻 code) Code
(🔣 data) Data
(📖 doc) Documentation
(🎨 design) Design
(🤔 ideas) Ideas & Planning
(🚇 infra) Infrastructure
(🚧 maintenance) Maintenance
(🔌 plugin) Plugin/utility libraries
(💬 question) Answering Questions
(👀 review) Reviewed Pull Requests
(🔧 tool) Tools
(⚠️ test) Tests
(✅ tutorial) Tutorials

Example: https://github.com/pyproj4/pyproj

**Comment by trexfeathers** [comment]
Ooof I'm torn. The output is nice. But this is another piece of architecture, and we have so many already! Getting a lot of splinters from sitting on this fence but still I can't decide...

**Comment by trexfeathers** [comment]
@SciTools/peloton: discussion suggests that we all like this in principle, but are nervous about the overhead it would add. @pp-mo points out that repo approachability is worth more than we sometimes recognise
